cmd/config/cmd/merge3.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/cmd/config/cmddocs/commands"
+	"sigs.k8s.io/kustomize/kyaml/kio/filters"
+)
+
+func GetMerge3Runner(name string) *Merge3Runner {
+	r := &Merge3Runner{}
+	c := &cobra.Command{
+		Use:     "merge3 --ancestor [ORIGINAL_DIR] --from [UPDATED_DIR] --to [DESTINATION_DIR]",
+		Short:   commands.Merge3Short,
+		Long:    commands.Merge3Long,
+		Example: commands.Merge3Examples,
+		RunE:    r.runE,
+	}
+	fixDocs(name, c)
+	c.Flags().StringVar(&r.ancestor, "ancestor", "",
+		"Path to original package")
+	c.Flags().StringVar(&r.fromDir, "from", "",
+		"Path to updated package")
+	c.Flags().StringVar(&r.toDir, "to", "",
+		"Path to destination package")
+
+	r.Command = c
+	return r
+}
+
+func Merge3Command(name string) *cobra.Command {
```

Comment:
Would you write a test for this -- you can use the ioutil to create a 3 temp dirs and populate them with some data.